pathspider/cmd/traceroute.py

```diff
@@ -137,15 +137,15 @@ def queue_feeder(cond, inputfile, ipqueue): #needs work, some stuff is unnecessa
         for line in fh:
             job = json.loads(line)
             if job['dip'] in seen_targets:
-                logger.info("This target has already had a job submitted, skipping.")
+                logger.debug("This target has already had a job submitted, skipping.")
```

Comment:
This line should be "info" as researchers should be alerted if their input file has not been pre-processed.